<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mount</class>
 <widget class="QWidget" name="Mount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>317</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="primaryLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="LeftLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="primaryScopeGroup">
           <property name="title">
            <string>Primary Telescope</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <property name="spacing">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Aperture (mm)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="primaryScopeApertureIN">
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Focal Length (mm)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="primaryScopeFocalIN">
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="guideScopeGroup">
           <property name="title">
            <string>Guide Telescope</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <property name="spacing">
             <number>3</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Focal Length (mm)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="guideScopeApertureIN">
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="guideScopeFocalIN">
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Aperture (mm)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Configurations</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="scopeConfigCombo">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Scope configuration index. You can define up to 6 different combinations of primary and secondary scopes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="scopeConfigNameEdit">
           <property name="toolTip">
            <string>Configuration label</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveB">
           <property name="toolTip">
            <string>Save telescope information in configuration file</string>
           </property>
           <property name="text">
            <string>Save Telescope Info</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="Coordinates">
         <property name="title">
          <string>Coordinates</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="toolTip">
             <string>Right Ascension</string>
            </property>
            <property name="text">
             <string>RA</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="raOUT">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_7">
            <property name="toolTip">
             <string>Declination</string>
            </property>
            <property name="text">
             <string>DEC</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="decOUT">
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="toolTip">
             <string>Azimuth</string>
            </property>
            <property name="text">
             <string>AZ</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="azOUT">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_8">
            <property name="toolTip">
             <string>Altitude</string>
            </property>
            <property name="text">
             <string>ALT</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="altOUT">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="toolTip">
             <string>Hour Angle</string>
            </property>
            <property name="text">
             <string>HA</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="haOUT">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_10">
            <property name="toolTip">
             <string>Local Sidereal TIme</string>
            </property>
            <property name="text">
             <string>LST</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="lstOUT">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="parkB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Park</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="unparkB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>UnPark</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="clearAlignmentModelB">
           <property name="text">
            <string>Clear Alignment Model</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="AutoPark">
         <property name="title">
          <string>Auto Park</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Park At:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="startupTimeEdit">
            <property name="displayFormat">
             <string>dd/MM/yy hh:mm</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="countdownLabel">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>26</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <property name="text">
             <string>00:00:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startTimerB">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="media-playback-start"/>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopTimerB">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="media-playback-stop"/>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="RightLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QPushButton" name="mountToolBoxB">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Mount Control</string>
         </property>
         <property name="icon">
          <iconset theme="draw-text">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="trackingGroup">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Tracking</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QPushButton" name="trackOnB">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:checked
{
background-color: maroon;
border: 1px outset;
font-weight:bold;
}</string>
            </property>
            <property name="text">
             <string>ON</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="trackOffB">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:checked
{
background-color: maroon;
border: 1px outset;
font-weight:bold;
}</string>
            </property>
            <property name="text">
             <string>OFF</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="LimitGroup">
         <property name="title">
          <string>Limits</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="horizontalSpacing">
           <number>3</number>
          </property>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="minAltLimit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="enableLimitsCheck">
            <property name="text">
             <string>Enable Limits</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="minAltLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Minimum telescope altitude limit. If the telescope is below this limit, it will be commanded to stop.</string>
            </property>
            <property name="text">
             <string>Min. Alt:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="maxAltLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximum telescope altitude limit. If the telescope is above this limit, it will be commanded to stop.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Max. Alt:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="maxAltLimit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <double>90.000000000000000</double>
            </property>
            <property name="value">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>line</zorder>
  <zorder></zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <tabstops>
  <tabstop>primaryScopeApertureIN</tabstop>
  <tabstop>primaryScopeFocalIN</tabstop>
  <tabstop>guideScopeApertureIN</tabstop>
  <tabstop>guideScopeFocalIN</tabstop>
  <tabstop>scopeConfigCombo</tabstop>
  <tabstop>scopeConfigNameEdit</tabstop>
  <tabstop>saveB</tabstop>
  <tabstop>mountToolBoxB</tabstop>
  <tabstop>trackOnB</tabstop>
  <tabstop>trackOffB</tabstop>
  <tabstop>raOUT</tabstop>
  <tabstop>decOUT</tabstop>
  <tabstop>azOUT</tabstop>
  <tabstop>altOUT</tabstop>
  <tabstop>haOUT</tabstop>
  <tabstop>lstOUT</tabstop>
  <tabstop>minAltLimit</tabstop>
  <tabstop>maxAltLimit</tabstop>
  <tabstop>parkB</tabstop>
  <tabstop>unparkB</tabstop>
  <tabstop>clearAlignmentModelB</tabstop>
  <tabstop>enableLimitsCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
